starter-kits/product/_details.md

```diff
@@ -1,19 +1,24 @@
 % ## Background
+% - optional
 
-% ## What this product offers
-
-% ## Data description
+% ## Product purpose
```

Comment:
This is just a suggestion but maybe we could reword this to put the 'active' word first:
`Product purpose` => `Purpose of product`

Reply:
These words are copied from the corporate tool, so we can't change them.